great_expectations/validator/validator.py

```diff
@@ -249,6 +249,10 @@ def validate_expectation(self, name: str):
 
         def inst_expectation(*args, **kwargs):
             # this is used so that exceptions are caught appropriately when they occur in expectation config
+
+            # TODO: JPC - THIS LOGIC DOES NOT RESPECT DEFAULTS SET BY USERS IN THE VALIDATOR VS IN THE EXPECTATION
+            # DEVREL has action to develop a new plan in coordination with MarioPod
+
```

Comment:
Should this be left in the code? And have we documented this outside of the codebase?

Reply:
Also, to @jcampbell for the first, though I think it may be worth leaving in. Yes to the second (though we should confirm timing).

Reply:
agreed leave in.